content/pages/security.md

```diff
@@ -7,6 +7,9 @@ template: security
 If you believe you have discovered a vulnerability in Solr, you may first want to consult the [list of known false positives](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) to make sure you are reporting a real vulnerability.
```

Comment:
```suggestion
Solr depends on lots of other open-source software -- "dependencies".  If a CVE is published (a publicly identified vulnerability) against one of them, the Solr project will review it to see if it's actually exploitable in Solr -- usually they aren't.  The non-exploitable vulnerabilities are published [on this Confluence page](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools).  If you don't see a CVE there, ask us about it.
```

Reply:
I like this, but I guess it begs the question where should they ask us. On the security@ or users@ list?

Reply:
security@ is reasonable / prudent, I think.